src/training/vision_eval.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import os
+
+import numpy as np
+import torch
+import tqdm
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix
+
+from joblib import Parallel, delayed
+
+
+def mapper(train_tensors, train_targets, val_tensors, val_targets, reg_value):
```

Comment:
resolved offline: let's try to keep everything in one file if possible for now, too much abstraction will cause us pain later.